### PR TITLE
LinusHomeworkFixed.md

### PR DESCRIPTION
**Pull request by Bahia369**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Review by mikinet — CHANGES_REQUESTED** [review]
Hi Bahia, I think this answer about JS in Q.9 is just fine, but you can make it even much better if you mention something special about JS. Cheers.

File: homework.md
Comment (jip1984) [inline homework.md:27]: Hypertext is one word (Just commenting for homework purposes)

File: homework.md
Comment (jip1984) [inline homework.md:73]: You make local code changes and then submit those changes to a remote project maintainer for review before those changes are implemented, or merged. This is called a pull request; you are requesting that someone reviews and approves your changes before they become final.

File: homework.md
Comment (samia-pk) [inline homework.md:77]: Git History log: it shows you all the history of changes in a file.

please leave feedback on my work also. thanks

File: homework.md
Comment (samia-pk) [inline homework.md:47]: JavaScript: it helips you to make your web page more interactive in various way.

please leave feedback on my work also. thanks

File: homework.md
Comment (samia-pk) [inline homework.md:75]: Git History log: it shows you all the history of changes in a file.

please leave feedback on my work also. thanks

**Pull request closed without merging**
